charmcraft.yaml

```diff
@@ -8,13 +8,13 @@ description: |
   Observability Stack (COS).
 subordinate: false
 
-bases:
-  - build-on:
-    - name: ubuntu
-      channel: "22.04"
-    run-on:
-    - name: ubuntu
-      channel: "22.04"
+parts:
+  charm:
+    plugin: charm
+    source: .
```

Comment:
I'm curious about why this was required? Strange that it worked without a parts definition at all... perhaps that's just the default definition?

Reply:
```
Bad charmcraft.yaml content:
- dictionary should have at least 1 item after validation, not 0 (in field 'parts')
```